roles/mirror_catalog/tasks/main.yml

```diff
@@ -56,6 +56,9 @@
       {% endif %}
       --max-components=3
```

Comment:
This is unrelated to your change, but considering the layout of the prega repo we will need to increase this to 4.

Reply:
I'll add a way to modify this then